<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HomeWindow</class>
 <widget class="QMainWindow" name="HomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>699</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>HomeWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>911</width>
      <height>661</height>
     </rect>
    </property>
    <property name="title">
     <string>Video Streams</string>
    </property>
    <widget class="QOpenGLWidget" name="openGLWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>421</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QOpenGLWidget" name="openGLWidget_3">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>40</y>
       <width>421</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QOpenGLWidget" name="openGLWidget_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>340</y>
       <width>421</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QOpenGLWidget" name="openGLWidget_2">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>340</y>
       <width>421</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnStartNew">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>40</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Start New</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalibrate">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>160</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Calibrate</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadVideo">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>290</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Load Video</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLoadTest">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>410</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Load Test</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCompare">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>530</y>
      <width>250</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Compare Tests</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
